browser/templates/browser/add_person.html

```diff
@@ -0,0 +1,135 @@
+{% extends "browser/change_base.html" %}
```

Comment:
why don't you reuse the change_person template? Doesn't that do the same thing?

Reply:
Reusing change_person template and deleted the add_person template